docs/pages/server-access/guides/auditd.mdx

```diff
@@ -0,0 +1,125 @@
+---
+title: Configure SSH with the Linux Auditing System
+description: How to configure Teleport SSH with auditd (Linux Auditing System).
+h1: Linux Auditing System (auditd)
+---
+
+You can configure Teleport's SSH Service to integrate with the Linux Auditing System (auditd).
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+- A running Teleport Node. See the [Server Access Getting Started Guide](../getting-started.mdx) for  how to add a Node to your Teleport cluster.
+- Linux kernel 2.6.6+ compiled with `CONFIG_AUDIT`. Most Linux distributions have this option enabled by default.
+- `auditctl` to check auditd status (optional).
+
+(!docs/pages/includes/tctl.mdx!)
+
+## Step 1/3. Check system configuration
+
+Teleport automatically sends audit events when it discovers that auditd is enabled in the system.
+You can verify that by calling `auditctl -s` as root.
+
+Here is an example output from that command:
+
+```bash
+$ sudo auditctl -s
+enabled 1
+failure 1
+pid 879
+rate_limit 0
+backlog_limit 8192
+lost 0
+backlog 0
+backlog_wait_time 60000
+backlog_wait_time_actual 0
+loginuid_immutable 0 unlocked
+```
+The first line `enabled 1` indicates that auditd is enabled, and Teleport will send events.
+
+Because of the Teleport distributed nature all events are generated on SSH node, where
+`invalid user` events are also generated on the Proxy service when a Teleport user fails to authenticate.
+
+## Step 2/3. Start Teleport
+
+It's important to run Teleport as a system service (systemd service, for example) with root permissions.
+Otherwise, Teleport won't send any events to auditd due to lack of permissions.
+
+<Admonition type="warning">
+
+  Make sure that the Teleport process has its login UID unset. Otherwise, a session ID won't be set correctly in the emitted events.
+  You can verify that by calling `cat /proc/$(pidof teleport)/loginuid`. The value should be set to 4294967295.
+
+</Admonition>
+
+## Step 3/3. Enable the PAM integration (optional)
+
+Auditd can generate additional events when PAM (Pluggable Authentication Modules) is enabled. To enable PAM integration
+in Teleport add the following `pam` section to options to the SSH service:
+
+```yaml
+ssh_service:
+  # Enabled SSH Service
+  enabled: true
+  # Enable PAM integration
+  pam:
+    # "no" by default
+    enabled: true
+    # use /etc/pam.d/sshd configuration (the default)
+    service_name: "sshd"
+```
+
+PAM-generated events depend on your `sshd` configuration when the integration is enabled. Most system generates events
+like `USER_ACCT` or `USER_START`. Additionally, TTY input can be logged by enabling `pam_tty_audit.so` module.
+
+For more details please refer to [PAM](./ssh-pam/#set-up-pam-on-a-linux-machine-running-teleport) or your operating system
+documentation.
+
+When PAM integration is enabled, audit events should closely match events generated by OpenSSH.
+
+## How to trace SSH session
```

Comment:
```suggestion
## Step 4/4. Trace SSH sessions with auditd
```
Suggested to:
- Use the standard step format for our how-to guides
- Link this section more explicitly to auditd so users know how it ties in with the rest of the guide